package.json

```diff
@@ -28,33 +28,33 @@
   "author": "Philipp Fromme",
   "license": "MIT",
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^25.0.7",
+    "@rollup/plugin-commonjs": "^25.0.8",
     "@rollup/plugin-node-resolve": "^15.2.3",
     "@rollup/plugin-terser": "^0.4.4",
     "chai": "^4.4.0",
     "diagram-js": "^14.0.0",
     "eslint": "^8.56.0",
     "eslint-plugin-bpmn-io": "^1.0.0",
-    "karma": "^6.4.2",
+    "karma": "^6.4.3",
     "karma-chrome-launcher": "^3.2.0",
-    "karma-firefox-launcher": "^2.1.2",
+    "karma-firefox-launcher": "^2.1.3",
     "karma-mocha": "^2.0.1",
     "karma-sinon-chai": "^2.0.2",
-    "karma-webpack": "^5.0.0",
-    "mocha": "^10.2.0",
+    "karma-webpack": "^5.0.1",
+    "mocha": "^10.4.0",
     "mocha-test-container-support": "^0.2.0",
-    "npm-run-all2": "^6.0.0",
-    "puppeteer": "^22.0.0",
-    "rollup": "^4.9.4",
-    "sinon": "^17.0.1",
+    "npm-run-all2": "^6.2.0",
+    "puppeteer": "^22.10.0",
+    "rollup": "^4.18.0",
+    "sinon": "^18.0.0",
     "sinon-chai": "^3.7.0",
-    "webpack": "^5.89.0"
+    "webpack": "^5.91.0"
   },
   "dependencies": {
     "hammerjs": "^2.0.8",
     "min-dash": "^4.0.0",
-    "min-dom": "^4.1.0",
```

Comment:
Let's not do this (yet) and mirror core modeling versions; fine to stick with the previous majors for min-dom and tiny-svg.